baselines/asset/src/v1/asset_service_client.ts.baseline

```diff
@@ -232,12 +237,13 @@ export class AssetServiceClient {
           throw err;
         });
 
+      const descriptor =
+        this.descriptors.longrunning[methodName] ||
+        undefined;
       const apiCall = this._gaxModule.createApiCall(
         callPromise,
         this._defaults[methodName],
-        this.descriptors.page[methodName] ||
-            this.descriptors.stream[methodName] ||
-            this.descriptors.longrunning[methodName]
+        descriptor
```

Comment:
was the bug simply that this `descriptor` value wasn't be passed into `createApiCall`?

Reply:
the `this.descriptors.batching[methodName]` was not passed.